README.md

```diff
@@ -173,6 +173,75 @@ loader
 
 If `options.resolve` is truthy, speccy will resolve _external_ references.
 
+### Adding Custom Rules
+
+You're able to create your own custom rules for Speccy. To accomplish this, follow these steps:
+
+1. Add new rule object to `rules/default.json`
+```json
+{
+    "name": "default-and-example-are-redundant",
+    "object": "*",
+    "enabled": true,
+    "description": "don't need to define an example if its exactly the same as your default",
+    "notEqual": ["default", "example"]
+}
+```
+
+2. Add rule logic code into `lib/linter.js`
+```javascript
+if (rule.notEqual) {
```

Comment:
This might be a fairly complex rule to show off, could you find something a bit shorter to convey the point, and mention the rule just needs to be a should.js assertion wrapped in ensure?

Reply:
I'll find a shorter example and make the the change